internal/boxcli/list.go

```diff
@@ -57,6 +62,28 @@ func listCmd() *cobra.Command {
 			return nil
 		},
 	}
+
+	cmd.Flags().BoolVar(&flags.outdated, "outdated", false, "List outdated packages")
 	flags.config.register(cmd)
 	return cmd
 }
+
+// outdatedPackage prints a list of outdated packages.
+func outdatedPackage(cmd *cobra.Command, box *devbox.Devbox) error {
+	results, err := box.Outdated(cmd.Context())
```

Comment:
nit: can we rename the api to `box.OutdatedPackages`?

this could be more clear as `nameToUpdateVersion` but don't feel strongly